tests/app/db.py

```diff
@@ -1075,7 +1075,7 @@ def create_broadcast_event(
         transmitted_areas=transmitted_areas or broadcast_message.areas,
         transmitted_sender=transmitted_sender or 'www.notifications.service.gov.uk',
         transmitted_starts_at=transmitted_starts_at,
-        transmitted_finishes_at=transmitted_finishes_at or datetime.utcnow(),
+        transmitted_finishes_at=transmitted_finishes_at or datetime.utcnow() + timedelta(hours=1),
```

Comment:
Optional: This is fine. Another option could be `transmitted_finishes_at=transmitted_finishes_at or transmitted_starts + timedelta(hours=24)` as that is likely what we will be having as real data

Reply:
its complicated, we dont currently set transmitted_starts as that's for if we want the CBC to delay the start of the msg which we currently don't do. but i bumped to `now + 24 hrs` to give it a more realistic buffer if you didn't specify it manually